runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheClientFetchFactory.java

```diff
@@ -913,6 +915,7 @@ private void doClientFanoutReplyWindow(
             this.deltaType = deltaType;
             this.isolation = isolation;
             this.isolatedOffset = isolation == READ_COMMITTED ? () -> group.stableOffset : () -> group.latestOffset;
+            this.targetOffset = isolation == READ_COMMITTED ? () -> initialGroupStableOffset : () -> initialGroupLatestOffset;
```

Comment:
Please rename `targetOffset` to `initialGroupIsolatedOffset` as this is the same condition as `isolatedOffset` but specific to initial group value.